country-dashboard/src/index.tsx

```diff
@@ -1,15 +1,31 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import { ChakraProvider } from "@chakra-ui/react";
+import CountryDetail from "./pages/CountryDetail";
+// routing
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <App />,
+  },
+  {
+    path: "CountryDeatail/:CountryName",
```

Comment:
We should use snake-case in url routes, not PascalCase. Also, don't use Country name in the URL, use code so there's no spaces. Spaces can mess up URLs. -1